_posts/2016-03-29-exploring-angular-1.5-lifecycle-hooks.md

```diff
@@ -36,6 +36,8 @@ Lifecycle hooks in Angular landed first in the Angular 2 alpha release and they
 
 However, this article is on Angular 1.5. Since Angular 1 is evolving in a way to keep the gap to Angular 2 as small as possible, some lifecycle callbacks have been backported to the current "best Angular yet". Let's take a look at them one by one.
 
+Note: The Angular lifecycle-hooks weren't introduced until version 1.5.3.
```

Comment:
Can we make this rather "positive" by saying it has been introduce in 1.5.3 ? :)